zeppelin-server/src/test/java/org/apache/zeppelin/AbstractZeppelinIT.java

```diff
@@ -18,7 +18,9 @@
 package org.apache.zeppelin;
 
 
+import com.amazonaws.util.Base64;
```

Comment:
Sure, I'll change this import, but this is something that I was about to ask as well;
https://github.com/apache/incubator-zeppelin/pull/719/files#diff-ee7042344a99f6984ef528dbbe372bc2R139
Is there any other way to take that file out from travis server ?